internal/constants/scripts.go

```diff
@@ -87,13 +89,17 @@ function find_my_mac() {
 
 function correlate_int_mac() {
     # Correlate the Mac with the interface
+    METRIC_KEY="route-metric="
+    METRIC_VAL=100
     for INTERFACE in $(ls -1 /sys/class/net | grep -v lo)
     do
         INT_MAC=$(cat /sys/class/net/${INTERFACE}/address)
         if [[ ! -z ${INT_MAC} ]]; then
             echo "MAC to check: ${INT_MAC}"
             find_my_mac ${INT_MAC}
             if [[ "${FOUND_MAC,,}" == "${INT_MAC,,}" ]]; then
+                export FOUND_ROUTE_METRIC=$METRIC_KEY$METRIC_VAL
+                let METRIC_VAL+=1
```

Comment:
Exporting it here is confusing. I think you should set the constants at the root level, and then set and increment in https://github.com/openshift/assisted-service/blob/09499d78c7d785ad7e7ecc7df2d69ef14c03c76b/internal/constants/scripts.go#L52 like we do with the rest of the values for a NIC.

Reply:
this patch is temporary, we are going to move the whole implementation of nmconnections to the nmstate lib, which will produce nmconnection files based on yamls